website/pages/en/cookbook/near.mdx

```diff
@@ -186,31 +186,33 @@ As a quick primer - the first step is to "create" your subgraph - this only need
 
 Once your subgraph has been created, you can deploy your subgraph by using the `graph deploy` CLI command:
 
-```
+```sh
 $ graph create --node <graph-node-url> subgraph/name # creates a subgraph on a local Graph Node (on the Hosted Service, this is done via the UI)
 $ graph deploy --node <graph-node-url> --ipfs https://api.thegraph.com/ipfs/ # uploads the build files to a specified IPFS endpoint, and then deploys the subgraph to a specified Graph Node based on the manifest IPFS hash
 ```
 
 The node configuration will depend on where the subgraph is being deployed.
 
-#### Hosted Service:
+### Hosted Service
 
-```
+```sh
 graph deploy --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ --access-token <your-access-token>
 ```
 
-#### Local Graph Node (based on default configuration):
+### Local Graph Node (based on default configuration)
```

Comment:
@benface This gave me an idea to enable another rule https://github.com/remarkjs/remark-lint/tree/main/packages/remark-lint-no-heading-punctuation

Reply:
@B2o5T – Good one!